README.md

```diff
@@ -70,8 +70,21 @@ And to deploy a node with the cirros image (image\_source above)
 You can also see the status of the bootkube.sh script which is running via
 `journalctl -b -f -u bootkube.service`.
 
-## 2) Cleanup
+## OCP cleanup & re-install
 
-To clean up your environment, run::
+To clean up the ocp deployment and re-install ocp, run::
+
+make dev-run
```

Comment:
The idea of these scripts is developers can re-run whatever bits they need, IMHO there's not going to be a single target which suits all developers, so I'd rather we just referenced the individual cleanup scripts here

Reply:
gotcha, I will remove this option, leave the clean command, but in the README I will reference the scripts as primary cleaning tools

Reply:
Sounds good, when you're done can you also pls squash the commits?  Then we're good to merge I think, thanks!